packages/cli/src/module-resolver.ts

```diff
@@ -1,12 +1,17 @@
 import { resolve } from '@ts-bridge/resolver';
 import type { FileSystemInterface, FileFormat } from '@ts-bridge/resolver';
 import chalk from 'chalk';
+import { init, parse } from 'cjs-module-lexer';
 import { resolve as resolvePath, extname } from 'path';
 import type { System } from 'typescript';
 import { pathToFileURL } from 'url';
 
 import { warn } from './logging.js';
 
+// This initialises `cjs-module-lexer` so that we can parse CommonJS modules
+// quickly using the WASM binary.
+await init();
```

Comment:
Any reason not to do this lazily without a top-level await?

Reply:
What would the benefit of that be?

Reply:
It would complicate a bunch of stuff too, since everything is synchronous right now, and TypeScript's API is completely synchronous.

Reply:
I guess my worry would be whether top-level await is supported everywhere we want to run ts-bridge. But that might not be an issue?

Reply:
The CLI always runs as ESM and requires a modern Node version that supports top-level await.